src/platform_impl/windows/system_tray.rs

```diff
@@ -217,6 +223,18 @@ unsafe extern "system" fn tray_subclass_proc(
     subclass_input.hmenu = Some(HMENU(wparam.0 as _));
   }
 
+  if msg == WM_USER_UPDATE_TRAYICON {
+    let icon = wparam.0 as *mut Icon;
+    subclass_input.icon = (*icon).clone();
```

Comment:
@wusyong does this correctly free the memory of the original icon? given that `subclass_input` is a raw pointer and it is only dropped when the app exists.

Reply:
from a quick test in rust playground it is dropped correctly.